hw/top_earlgrey/data/ip/chip_edn_testplan.hjson

```diff
@@ -30,15 +30,29 @@
             - Enable ENTROPY_SRC in boot-time mode, e.g., by setting CONF.FIPS_ENABLE to kMultiBitBool4False.
             - Enable CSRNG.
             - Enable EDN1 (the one connected to OTBN RND) in boot-time request mode.
+            - Disable ENTROPY_SRC. Since EDN1 provides entropy only to the OTBN, the CSRNG will be idle.
```

Comment:
Nit: one sentence per line please:
```
- Disable ENTROPY_SRC.
  Since EDN1 provides entropy only to the OTBN, the CSRNG will be idle.
```
Similar below.

Reply:
Nit: the CSRNG -> CSRNG (remove the definite article), the same should be done below.

Reply:
Thanks, this will be fixed in the next version of this PR.